items.json

```diff
@@ -128387,19 +128387,19 @@
         "thrustSpeed": 87,
         "swingDamage": 28,
         "swingDamageType": "Blunt",
-        "swingSpeed": 87
+        "swingSpeed": 86
       }
     ]
   },
   {
-    "id": "crpg_onehanded_gada_v1_h0",
-    "baseId": "crpg_onehanded_gada_v1",
+    "id": "crpg_onehanded_gada_v2_h0",
```

Comment:
example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.